style.css

```diff
@@ -623,12 +623,99 @@ form {
   #about-myself {
     padding-inline: 100px;
   }
+
+  .mobile-menu {
+    display: none;
+  }
+
+  .close-button {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
 }
 
 @media (max-width: 768px) and (max-width: 1100px) {
   .page_links {
     display: none;
   }
+
+  .mobile-menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background: #6070ff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(8px);
```

Comment:
- [ ] backdrop-filter is not available on firefox and it is causing the mobile menu background to differ from the design on firefox as shown below:
- Your page
![image](https://user-images.githubusercontent.com/61377322/140860766-e58c54bb-d4d8-4c73-91c0-6e8bd17caa9f.png)

- Figma design
![image](https://user-images.githubusercontent.com/61377322/140860786-1ff0231a-dc0d-41c5-a7c1-83a5c24edf04.png)

> Hint: When the mobile menu is active, you can set the filter property for the background elements as `filter: blur(4px);` to achieve the same result. Kindly refer [here](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter) for more.

Reply:
Please give me more insights on this. I have failed to find a solution.